__tests__/server.test.js

```diff
@@ -72,6 +72,78 @@ describe("/api/articles/:article_id", () => {
         expect(response.body.message).toBe("Article does not exist");
       });
   });
+  test("PATCH 200: Should return an updated article object with the votes increased by the passed amount", () => {
+    const updatedVotes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updatedVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("article_id");
```

Comment:
It would be good to be more specific here, to check the right article has been updated